### PR TITLE
added two files and updated READme

### PR DESCRIPTION
**Pull request by fangshing4**

### CODE REVIEW

**Review by Evans237 — APPROVED** [review]
1.

**Comment by Evans237** [comment]
carry on boss

**Review by Ricky8484 — APPROVED** [review]
good to go.

**Review by langsmith2010 — COMMENTED** [review]
Great job guys keep it up

**Review by langsmith2010 — APPROVED** [review]

File: README.md
Comment (Evans237) [inline README.md:1]: check

File: provider.tf
Comment (langsmith2010) [inline provider.tf:1]: I guess you should add the Main.tf as well.

**Pull request merged**